### PR TITLE
Update install-gpu-model-builder.md

### PR DESCRIPTION
**Pull request by matt-gribben**
## Summary

Updated GPU instructions so they don't depend on the user having NVIDIA software install and follow Microsoft guidance in learning centre on how to find GPU information (https://www.microsoft.com/en-us/windows/learning-center/how-to-check-gpu)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/machine-learning/how-to-guides/install-gpu-model-builder.md](https://github.com/dotnet/docs/blob/b3f588c2f1ea158545ccae33871e16432ceafa3b/docs/machine-learning/how-to-guides/install-gpu-model-builder.md) | [How to install GPU support in Model Builder](https://review.learn.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-gpu-model-builder?branch=pr-en-us-40824) |

### CODE REVIEW

**Comment by matt-gribben** [comment]
@dotnet-policy-service agree

**Comment by matt-gribben** [comment]
@luisquintanilla Do you need anything else from me before you can review this?

**Review by luisquintanilla — APPROVED** [review]
LGTM. Thanks @matt-gribben

File: docs/machine-learning/how-to-guides/install-gpu-model-builder.md [hunk 1]
```diff
@@ -41,13 +41,20 @@ However, if local GPUs or Azure are not an option for you, these scenarios also
...
-1. You will see the name of your NVIDIA GPU
+***Check GPU from Settings***
 
-**I don't see NVIDIA Control Panel (or it fails to open) but I know I have an NVIDIA GPU.**
+1. Right-click on the Windows start menu icon and select **Settings**.
+1. Select **Settings** > **System**
+1. Select **Display** and scroll down to **Related settings**.
+1. Select **Advanced display**. Your GPU’s make and model should be shown under **Display information**.
+
+***Check GPU from Task Manager***
```
Comment (BillWagner) [inline docs/machine-learning/how-to-guides/install-gpu-model-builder.md:51]: adding a blank line for the lint issue:

```suggestion
***Check GPU from Task Manager***

```

**Pull request merged**